templates/service-template.yml

```diff
@@ -1085,14 +1080,9 @@ objects:
             - --observatorium-timeout=${OBSERVATORIUM_TIMEOUT}
             - --observatorium-auth-type=${OBSERVATORIUM_AUTH_TYPE}
             - --observatorium-token-file=/secrets/service/observatorium.token
-            - --observability-red-hat-sso-logs-client-id-file=/secrets/observatorium/logs.clientId
```

Comment:
If we go ahead with the removal of flags I understand some changes on the Makefile might be needed too. Has that been reviewed?
The same question applies to the secrets templates